core/cas-server-core-logging/src/main/java/org/apereo/cas/logging/web/ThreadContextMDCServletFilter.java

```diff
@@ -71,6 +75,14 @@ public void doFilter(final ServletRequest servletRequest, final ServletResponse
             addContextAttribute("scheme", request.getScheme());
             addContextAttribute("timezone", TimeZone.getDefault().getDisplayName());
 
+            if (StringUtils.isNotBlank(casProperties.getAudit().getAlternateClientAddrHeaderName())) {
```

Comment:
Excellent.

Rather than tying "audit" functionality with logging, and rather than introducing individual fields, wouldn't it be better if you simply included all request headers in MDC and then picked out anything you needed in the logging configuration instead of documenting and selecting individual headers?

Reply:
That's a great idea. I'll work on that. Thanks for the feedback.

Reply:
Wonderful. Please ping back when you have made the change and we'll be happy to merge. Thank you for picking this up!